README.md

```diff
@@ -103,12 +134,12 @@ npm test
 ./run-tests.sh
 ````
 
-#### Options
+### Options
```

Comment:
Do these apply to local-dev also? May want to add them if not.

Reply:
Originally it was a subsection of Testing and I will make it such.

Reply:
Never mind, it is a subsection....hmm...good question....

Reply:
I don't know, I am leaving it as is.